modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -129,22 +129,26 @@ Stack Overflow. Max stack is 256
 
 You can configure the maximum stack size by using the property `com.mulesoft.dw.stacksize`.
 
-== DataWeave Warnings
+=== No space left on device
+In order to handle large payloads, DataWeave generates some data which is handled in memory unless it exceeds a certain configurable limit.
+When that happens the data is stored on disk as output, input, and buffer files in a temporary directory. See xref:dataweave-memory-management.adoc[DataWeave Memory Management] for further details.
 
-The following are some common DataWeave warnings that you can find in your dump file, along with some details to address these warnings.
+Once the streams referencing those files are closed the files are released. The closure normally happens when flows complete their execution, so long-running and particularly concurrent executions could involve having many of these files living at once.
 
-=== End of Input Was Reached
+To avoid the exception `No space left on device` more resources are needed to be able to run the app, or the app needs to be reviewed to reduce its resource consumption.
 
-This warning shows because DataWeave treats the Unicode non-character `U+FFFF` as a special character that indicates the end of input.
+Sometimes bugs can lead to them not being removed even after the flow has completed.
+If this case isn't fixed in the latest update, there’s no way to work around the issue, and this can be reported to the Mulesoft support team.
```

Comment:
```suggestion
Such cases require an update to the code. If the latest Mule release does not fix this issue, please report the issue to the MuleSoft support team.
```

Reply:
Added `If the latest Mule release does not fix this issue, please report the issue to the MuleSoft support team.`